pythainlp/translate/core.py

```diff
@@ -0,0 +1,160 @@
+# -*- coding: utf-8 -*-
+import os
+import tarfile
+from collections import defaultdict
+from functools import partial
+
+from pythainlp.corpus import download, get_corpus_path
+from pythainlp.tools import get_full_data_path, get_pythainlp_data_path
+from fairseq.models.transformer import TransformerModel
+from sacremoses import MosesTokenizer, MosesDetokenizer
+from pythainlp.tools import get_full_data_path, get_pythainlp_data_path
+from pythainlp.tokenize import word_tokenize as th_word_tokenize
+
+en_word_detokenize = MosesDetokenizer("en")
+en_word_tokenize = MosesTokenizer("en")
+th_word_tokenize = partial(th_word_tokenize, keep_whitespace=False)
+
+
+def download_model():
+    print("Download model ...")
```

Comment:
Will this be printed every time when importing the module?

Reply:
> 
> 
> Will this be printed every time when importing the module?

Only first time importing the module.